src/design/typography/demo/figure.twig

```diff
@@ -0,0 +1,4 @@
+<figure>
+  <img src="https://picsum.photos/600/300">
+  <figcaption>I'm a picture with a caption.</figcaption>
```

Comment:
This is a _total_ nit, but from one design nerd to another: Maybe swap the apostrophe for a curly single quote in content? (`’`)

Reply:
Chiming in as a dev nerd, I wonder if we could create some sort of helper (or whatever the Twig equivalent is) that prettifies text like this. For example [markdown-it](https://github.com/markdown-it/markdown-it) has a `typographer` option that enables "some language-neutral replacement + quotes beautification"

Though that maybe makes more sense on the WP side of things?

Reply:
We get this for free in WP land, I'm just being nitpicky. I don't think it's worth justifying another dependency.